views/index.html

```diff
@@ -21,14 +21,16 @@ <h1 style="color:white;">
         </h1>
         <form uk-sticky="animation: uk-animation-slide-top; offset: 10; show-on-up: true;">
           <input id="mah-nam" type="text" maxlength="100" placeholder="aye, sir!" required>
-          <button type="submit">coolify</button>
+          <button id='coolify-submit' type="submit">coolify</button>
```

Comment:
there are 2 unique functions defined in `public/client.js`, which are executed on `click` for each button.